app/index.html

```diff
@@ -255,6 +256,10 @@
               <!-- <iron-icon icon="assessment"></iron-icon> -->
               <span>ALL PROJECTS</span>
             </p>
+            <p class='menu-item' id='news'>
+              <!-- <iron-icon icon="assessment"></iron-icon> -->
```

Comment:
Commented out code is very bad practice as it introduces a lot of confusion for reader. I would remove it, especially that is very simple piece of code.